scripts/next-data/generateNodeReleasesData.mjs

```diff
@@ -0,0 +1,53 @@
+import nodevu from '@nodevu/core';
+import { writeFile } from 'fs/promises';
+import { join } from 'path';
+
+import { getRelativePath } from './_helpers.mjs';
+
+const generateNodeReleasesData = async () => {
+  const [nodevuOutput, indexJsonOutput] = await Promise.all([
+    nodevu(),
+    // nodevu doesn't return release date and modules
+    // So it is a temporary workaround
+    (await fetch('https://nodejs.org/dist/index.json')).json(),
```

Comment:
Nodevu does have release date, yes.

Reply:
It's kinda stupid to fetch the same file two times (as that's what nodevu does, fetching the same file).

At best you can make a PR to nodevu instead of this tricky hack... 😅 

We don't need to rush to merge this PR, hence I rather not having this hack here.

Reply:
TBH I feel the same...

I was thinking of having the hack first because I don't want to block many ongoing PRs for migrating the download components. So after merging, I'll raise another PR to `nodevu` to add the missing fields.

If the consensus is okay for waiting, I don't mind. 😄 
I can draft the PR in the meantime.

Reply:
> I was thinking of having the hack first because I don't want to block many ongoing PRs for migrating the download components. So after merging, I'll raise another PR to nodevu to add the missing fields.

I don't mind that we block PRs, again, we have no rush in landing this. Let's do the development on its due time. I rather that we do it right already at the first iteration, as this reduces the complexity of our systems.

Reply:
> FYI the date in support.phases.dates.start is not equal to the release date in index.json.
It seems like they are a few days apart (1-2 days maybe?).

I would love to tag @bnb here but I'm not sure if they're currently banned on this repository :(

Reply:
@HinataKah0 regardless feel free to make a PR unto nodevu, I'm sure Tierney would be more than happy with the contribution!

Reply:
> [This](https://github.com/cutenode/nodevu) should be the correct nodevu repo, right?

yes

Reply:
Adding link to PR here: https://github.com/cutenode/nodevu/pull/32